pkg/vsphere/tasks/waiter.go

```diff
@@ -110,5 +111,7 @@ func isTaskInProgress(err error) bool {
 		}
 	}
 
+	//if none are found this is an unknown fault so we are dumping the structure.
+	log.Debugf("unexpected fault on task retry : %s", spew.Sdump(err))
```

Comment:
Suggest you omit ChangeVersion Conflict errors as well - those are expected. See lib/portlayer/exec/commit.go